golos.vesting/golos.vesting.cpp

```diff
@@ -65,7 +65,7 @@ void vesting::on_transfer(name from, name to, asset quantity, std::string memo)
     asset converted = token_to_vesting(quantity, *vesting, -quantity.amount);
     table_vesting.modify(vesting, name(), [&](auto& item) {
         item.supply += converted;
-        // TODO Add notify about supply changes #548
+        send_quantity_event(item.supply);
```

Comment:
Sending only quantities does not carry any information to the recipients of the event. Problem: recipient doesn't know which field was changed. See `cyber.token` contract for implementation example.